modules/space/rendering/renderablefluxnodes.cpp

```diff
@@ -0,0 +1,1106 @@
+    , _fluxColorAlphaIlluminance(FluxColorAlphaIlluminanceInfo, 1.f, 0.f, 1.f)
+    , _thresholdFlux(ThresholdFluxInfo, -1.5f, -50.f, 10.f)
+    , _filteringLower(FilteringInfo, 0.f, 0.f, 5.f)
+    , _filteringUpper(FilteringUpperInfo, 5.f, 0.f, 5.f)
+    , _amountofNodes(AmountofNodesInfo, 1, 1, 100)
+    , _defaultNodeSkip(DefaultNodeSkipInfo, 1, 1, 100)
+    , _earthNodeSkip(EarthNodeSkipInfo, 1, 1, 100)
+    , _fluxNodeskipThreshold(FluxNodeskipThresholdInfo, 0, -20, 10)
+    , _radiusNodeSkipThreshold(RadiusNodeSkipThresholdInfo, 0.f, 0.f, 5.f)
+    , _earthdistGroup({ "Earthfocus" })
+    , _distanceThreshold(DistanceThresholdInfo, 0.0f, 0.0f, 1.0f)
+    , _misalignedIndex(MisalignedIndexInfo, 0, -5, 20)
+    , _flowColor(
+        FlowColorInfo,
+        glm::vec4(0.96f, 0.88f, 0.8f, 0.5f),
+        glm::vec4(0.f),
+        glm::vec4(1.f)
+    )
+    , _flowEnabled(FlowEnabledInfo, false)
+    , _interestingStreamsEnabled(InterestingStreamsInfo, false)
+    , _flowGroup({ "Flow" })
+    , _flowParticleSize(FlowParticleSizeInfo, 5, 0, 500)
+    , _flowParticleSpacing(FlowParticleSpacingInfo, 60, 0, 500)
+    , _flowSpeed(FlowSpeedInfo, 20, 0, 1000)
+    , _useFlowColor(UseFlowColorInfo, false)
+    , _scaleFactor(TempInfo1, 150.f, 1.f, 500.f)
+    , _maxNodeDistanceSize(MaxNodeDistanceSizeInfo, 1.f, 1.f, 10.f)
+    , _nodeDistanceThreshold(NodeDistanceThresholdInfo, 0.f, 0.f, 40.f)
+    , _cameraPerspectiveEnabled(CameraPerspectiveEnabledInfo, false)
+    , _drawingCircles(DrawingCirclesInfo, false)
+    , _cameraPerspectiveGroup({" CameraPerspective"})
+    , _drawingHollow(DrawingHollowInfo, false)
+    , _gaussianAlphaFilter(GaussiandAlphaFilterInfo, false)
+    , _radiusPerspectiveEnabled(RadiusPerspectiveEnabledInfo, true)
+    , _perspectiveDistanceFactor(PerspectiveDistanceFactorInfo, 2.67f, 1.f, 20.f)
+    , _maxNodeSize(MaxNodeSizeInfo, 30.f, 1.f, 200.f)
+    , _minNodeSize(MinNodeSizeInfo, 2.f, 1.f, 10.f)
+    , _pulseEnabled(pulseEnabledInfo, false)
+    , _gaussianPulseEnabled(gaussianPulseEnabledInfo, false)
+    , _pulseAlways(AlwaysPulseInfo, false)
+{
+
+    const Parameters p = codegen::bake<Parameters>(dictionary);
+
+    _transferFunction = 
+        std::make_unique<TransferFunction>(p.colorTablePaths.standard);
+    _transferFunctionCMR =
+        std::make_unique<TransferFunction>(p.colorTablePaths.cmr);
+    _transferFunctionEarth =
+        std::make_unique<TransferFunction>(p.colorTablePaths.earth);
+    _transferFunctionFlow = 
+        std::make_unique<TransferFunction>(p.colorTablePaths.flow);
+    
+    _colorTablePath = p.colorTablePaths.standard;
+
+    _binarySourceFolderPath = p.sourceFolder;
+    if (std::filesystem::is_directory(_binarySourceFolderPath)) {
+        // Extract all file paths from the provided folder
+        _binarySourceFiles.clear();
+        namespace fs = std::filesystem;
+        for (const fs::directory_entry& e : fs::directory_iterator(
+            _binarySourceFolderPath)) {
```

Comment:
This should be on the previous line and if it doesn't fit, the entire `fs::directory_iterator(...)` should be on the next line and the `{` on yet another line